src/main/java/org/ihtsdo/rvf/core/service/DroolsRulesValidationService.java

```diff
@@ -73,11 +72,8 @@ public class DroolsRulesValidationService {
 
 	@PostConstruct
 	public void init() {
-		AmazonS3 anonymousClient = AmazonS3ClientBuilder.standard()
-				.withRegion(awsRegion)
-				.withCredentials(new AWSStaticCredentialsProvider(new AnonymousAWSCredentials()))
-				.build();
-		testResourceManager = new ResourceManager(testResourceConfig, new SimpleStorageResourceLoader(anonymousClient));
+		S3Client s3Client = S3Client.builder().credentialsProvider(ProfileCredentialsProvider.create()).build();
```

Comment:
We need to update here because we don't use profile and no profile name is given here.